examples/jsm/tsl/display/LensflareNode.js

```diff
@@ -0,0 +1,155 @@
+import { Color, RenderTarget, Vector2 } from 'three';
+import { convertToTexture, TempNode, nodeObject, Fn, NodeUpdateType, QuadMesh, PostProcessingUtils, NodeMaterial, passTexture, uniform, uv, vec2, vec3, vec4, max, float, sub, int, Loop, fract, pow, distance } from 'three/tsl';
+
+const _quadMesh = /*@__PURE__*/ new QuadMesh();
+const _size = /*@__PURE__*/ new Vector2();
+let _rendererState;
+
+/**
+ * References:
+ * https://john-chapman-graphics.blogspot.com/2013/02/pseudo-lens-flare.html
+ * https://john-chapman.github.io/2017/11/05/pseudo-lens-flare.html
+ */
+class LensflareNode extends TempNode {
+
+	static get type() {
+
+		return 'LensflareNode';
+
+	}
+
+	constructor( textureNode, ghostTintNode ) {
+
+		super();
+
+		this.textureNode = textureNode;
+
+		this.downSampleRatio = 4;
+		this.ghostTint = ghostTintNode;
+
+		this.updateBeforeType = NodeUpdateType.FRAME;
+
+		// render targets
+
+		this._renderTarget = new RenderTarget( 1, 1, { depthBuffer: false } );
+		this._renderTarget.texture.name = 'LensflareNode';
+
+		// uniforms
+
+		this.threshold = uniform( 0.5 );
```

Comment:
I would like to know if we can adopt a standard in the properties of post-processing classes. I think that adding `uniforms` and then modifying them using `flarePass.threshold.value` can be confusing for a node approach.

I think the approach used in `OutlineNode` should be used as a template.

This way we won't have any confusion if a user wants to replace the uniform node with a procedural value and we won't use uniform as default values ​​any more, just constants, which will help performance.

Reply:
So instead of having `threshold` it would be the following?
```js
this.thresholdNode = float( 0.5 );
```

Reply:
Updated to the new approach!